unit/goto-symex/symex_assign.cpp

```diff
@@ -156,9 +152,7 @@ SCENARIO(
         WHEN("foo is assigned a second time")
         {
           const exprt rhs2 = from_integer(1841, int_type);
-          exprt full_lhs2 = nil_exprt{};
-          full_lhs.type() = int_type;
-          symex_assign.assign_symbol(ssa_foo, full_lhs2, rhs2, guard);
+          symex_assign.assign_symbol(ssa_foo, expr_skeletont{}, rhs2, guard);
```

Comment:
I know I'm late to the party, but how is this equivalent to the previous code? The previous code had a type assigned. @romainbrenguier ?